assets/src/tinymce/web-stories.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { render } from '@wordpress/element';
+import { dispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import '../../../node_modules/@wordpress/components/build-style/style.css';
+import name from './store/name';
+import WebStoryMCEStore from './store';
+import WebStoriesModal from './containers/Modal';
+import { webStoriesData } from './utils/globals';
+import { setFields } from './utils/';
+
+const { _ } = window;
+
+/**
+ * Add button to tinyMCE editor.
+ */
+(function () {
```

Comment:
No need for this IIFE, as our code is not in global scope.

Reply:
Removed IIFE.